index.html

```diff
@@ -126,7 +126,7 @@ <h2 class="about_primary_text">About me</h2>
                         <textarea name="textarea" id="textarea" maxlength="500" required>Hey I’d like to connect to...</textarea>
```

Comment:
[OPTIONAL] Good job 💯  but instead of a text I'd suggest using a `placeholder` here, that gives the user an idea of how the message could be without having a predefined message. 🤷🏽